docs/release-notes.md

```diff
@@ -32,6 +32,21 @@ updates as well.
   2.9.10 has an infinite loop in a certain end-of-file situation." Fix
   by updating a dependency in documentation generation.
 
+### Zephyr-Rtos compatibility
+- for compatibility with zephyr 2.2.0 need to revert following commits:  
+    f4a9a9d  
+    470e2f3 (conflict: accept current changes)  
+    1084100  
+    41c5d50  
+    fefdea20  
+    8ecb89b  
+
+- zephyr-rtos is using mcuboot exact version reference in its manifest. You
+  might use any of zephyr running-master version which references the release and
+  very probably most of zephyr version chronologically around the release linked
+  one (the lower limit is
+  https://github.com/zephyrproject-rtos/zephyr/commit/0f9b362).
```

Comment:
My understanding of this sentence is that any Zephyr `master` after the given commit should work, is that what you mean?

Reply:
yes, any unless its will lost compatibility gain :(

Reply:
Can't you just mention something like

> Any Zephyr `master` after https://github.com/zephyrproject-rtos/zephyr/commit/0f9b362 should be able to work.

Reply:
I like Fabio's suggestion and I think it's also useful to emphasize that those zephyr versions that reference the 1.6.0 release probably work and can be used.